README.adoc

```diff
@@ -44,13 +44,13 @@ To use the development version instead, refer to the <<development-quickstart,De
 
 This section documents the Antora extensions that are provided by this library and how to configure them.
 
-IMPORTANT: Be sure to register each extension under the `antora.extensions` key in the playbook, not the `asciidoc.extensions` key.
+IMPORTANT: Ensure you register each extension under the `antora.extensions` key in the playbook, not the `asciidoc.extensions` key.
 
 === Algolia indexer
 
 This extension generates an Algolia index for each version of each component. The index entries are then saved to Algolia using the `saveObjects()` method, and also saved as JSON files in the site catalog. JSON files are published to the site root using the following template: `algolia-<component>-<version>.json`.
 
-NOTE: Only pages that include an `<article>` element with the `doc` class are indexed. Pages marked as "noindex" for "robots" are skipped.
+NOTE: Only pages that include an `<article>` element with the `doc` class are indexed.
```

Comment:
not the case anymore?

Reply:
No, we had that issue with beta versions. We add 'no-index' to beta versions so Google doesn't grab them since they're ephemeral. We removed the skip for those because their mainly for search engines rather than our internal search.